jsettlers.mapcreator/src/jsettlers/exceptionhandler/ErrorIcon.java

```diff
@@ -0,0 +1,58 @@
+package jsettlers.exceptionhandler;
```

Comment:
Please do not re-invent the wheel.

There is an eclipse project to do this (RelEng Tool).